OMCompiler/Compiler/Template/CodegenC.tpl

```diff
@@ -1970,7 +1970,7 @@ template functionInitialLinearSystemsTemp(list<SimEqSystem> linearSystems, Strin
            linearSystemData[<%ls.indexLinearSystem%>].equationIndex = <%ls.index%>;
            linearSystemData[<%ls.indexLinearSystem%>].size = <%size%>;
            linearSystemData[<%ls.indexLinearSystem%>].nnz = <%nnz%>;
-           linearSystemData[<%ls.indexLinearSystem%>].method = 1;
+           linearSystemData[<%ls.indexLinearSystem%>].method = 1;   /* Symbolic Jacobian avaialble */
```

Comment:
typo: available

Reply:
Thanks for finding. I'll update it.

Reply:
I really appreciate these kind of comments 👍